components/src/dynamo/vllm/omni/utils.py

```
@@ -12,13 +13,58 @@
+
+    try:
+        from vllm_omni.engine.async_omni_engine import load_and_resolve_stage_configs
+
+        _, stage_configs = load_and_resolve_stage_configs(
+            model,
+            stage_configs_path,
+            {},
+            default_stage_cfg_factory=None,
+        )
+        return stage_configs
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)